crates/cli/src/subcommands/login.rs

```diff
@@ -0,0 +1,169 @@
+use crate::Config;
+use clap::{Arg, ArgAction, ArgMatches, Command};
+use reqwest::Url;
+use serde::Deserialize;
+use webbrowser;
+
+pub fn cli() -> Command {
+    Command::new("login")
+        .arg(
+            Arg::new("host")
+                .long("host")
+                .default_value("https://spacetimedb.com")
+                .help("Fetch login token from a different host"),
+        )
+        .arg(
+            Arg::new("spacetimedb-token")
+                .long("token")
+                .conflicts_with("host")
+                .conflicts_with("new")
+                .help("Bypass the login flow and use a login token directly"),
+        )
+        .arg(
+            Arg::new("refresh-cache")
+                .long("refresh-cache")
+                .action(ArgAction::SetTrue)
+                .help("Clear the cached tokens and re-fetch them"),
```

Comment:
Tokens? Does this apply to the website session token or the spacetimedb token?

Reply:
It clears both. It was useful for me while testing since we kept updating server-side stuff, but I'm not particularly attached to any specific behavior (or necessarily keeping this at all).